benchmarks/benchmark_serving.py

```diff
@@ -652,6 +656,19 @@ def main(args: argparse.Namespace):
             raise ValueError(f"Unknown dataset: {args.dataset_name}") from err
     goodput_config_dict = check_goodput_args(args)
 
+    # Collect the sampling parameters.
+    sampling_params = {
+        k: v
+        for k, v in {
+            'top_p': args.top_p,
+            'top_k': args.top_k,
+            'min_p': args.min_p,
+            'temperature': args.temperature
```

Comment:
Shouldn't we remove the default temperature value that appears in `async_request_openai_completions` and `async_request_openai_chat_completions`?
Since it gets replaced anyway every time, I feel it hurts readability a bit.

Reply:
Thanks for the review @NickLucche ! I think keeping the default temperature in `backend_request_func.py` might be better because this file is likely used by other modules as well and not just by `benchmark_serving.py`, so we probably shouldn't modify its default behavior.